packages/core/src/internal-types.ts

```diff
@@ -18,6 +18,15 @@ export interface ParsedRequest extends ServerRequest {
 }
 
 export type FindRoute = (request: ParsedRequest) => ResolvedRoute<string>;
+/**
+ * Invokes method which is defined in the Application Controller
+ *
+ * @param controller Name of End user's application controller
```

Comment:
"Invokes method which is defined in the Application Controller"
s/be "Invokes a method defined in the Application Controller"

"End user's" s/be "end-user's"